morseMessenger/morseFunctions.tsx

```diff
@@ -1,10 +0,0 @@
-import { morseDictionary, DictOptions } from './morseDictionary';
-
-export function translateToMorse(sentence: string) {
```

Comment:
I'd keep it in this file as:
- it's nice to keep separated functions which don't depend on `state` in separate util files
- it's easier to unit test this function when it's in a separate file